sound/soc/sof/loader.c

```diff
@@ -279,6 +302,9 @@ static int snd_sof_fw_ext_man_parse(struct snd_sof_dev *sdev,
 		case SOF_EXT_MAN_ELEM_DBG_ABI:
 			ret = ext_man_get_dbg_abi_info(sdev, elem_hdr);
 			break;
+		case SOF_EXT_MAN_ELEM_CONFIG_DATA:
+			ret = ext_man_get_config_data(sdev, elem_hdr);
+			break;
```

Comment:
This is a useful thing to add even if we don't add full implementation. If we don't have the type in this switch case, we'll emit a a warning. If we get a lot of these, I think we need to make the warning on L309 a debug one. We have other means in place to check the fw version.